app/controllers/application_controller.rb

```diff
@@ -1,3 +1,16 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
```

Comment:
I haven't seen this security line before, I'll have to dig around and find this :exception method when I pull these changes down.